images/kcp-registrar/register.sh

```diff
@@ -20,16 +20,81 @@ set -o pipefail
 
 usage() {
 
-    printf "Usage: KCP_ORG="root:pipelines-service" KCP_WORKSPACE="compute" KCP_SYNC_TAG="release-0.5" DATA_DIR="/workspace" ./register.sh\n\n"
-
-    # Parameters
-    printf "The following parameters need to be passed to the script:\n"
-    printf "KCP_ORG: the organization for which the workload clusters need to be registered, i.e.: root:pipelines-service\n"
-    printf "KCP_WORKSPACE: the name of the workspace where the workload clusters get registered (created if it does not exist), i.e: compute\n"
-    printf "KCP_SYNC_TAG: the tag of the kcp syncer image to use (preset in the container image at build time and leveraged by the PipelineRun)\n"
-    printf "DATA_DIR: the location of the cluster files\n"
-    printf "          a single file with extension kubeconfig is expected in the subdirectory: credentials/kubeconfig/kcp\n"
-    printf "          kubeconfig files for compute clusters are expected in the subdirectory: credentials/kubeconfig/compute \n\n"
+    printf "
+Usage:
+    %s [options]
+
+Deploy Pipelines Service on the clusters as per the configuration in WORKSPACE_DIR.
+
+Mandatory arguments:
+    --kcp-org KCP_ORG
+        Organization for which the workload clusters need to be registered.
+        Example: 'root:pipelines-service'.
+        Can be set through \$KCP_ORG.
+    --kcp-workspace KCP_WORKSPACE
+        Name of the workspace where the workload clusters get registered (created if it
+        does not exist).
+        Example: 'compute'.
+        Can be set through \$KCP_WORKSPACE.
+    --kcp-sync-tag KCP_SYNC_TAG
+        Tag of the kcp syncer image to use (preset in the container image at build time
+        and leveraged by the PipelineRun).
+        Example: 'v0.6.0-alpha0'
+        Can be set through \$KCP_SYNC_TAG.
+
+Optional arguments:
+    -w, --workspace-dir WORKSPACE_DIR
```

Comment:
You need to be careful with such changes. Besides the fact that it has nothing to do with the stated PR goal and that it is disputable to see that as an "improvement" this is technically breaking the API.
DATA_DIR is referenced in multiple places:
- docs/kcp-registration.md
- gitops/sre/.tekton/kcp-registration.yaml
- gitops/pac/.tekton/kcp-registration.yaml
- DEVELOPMENT.md